### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by RomanHrytsiuk**
https://RomanHrytsiuk.github.io/layout_calendar/
https://RomanHrytsiuk.github.io/layout_calendar/report/html_report/

### CODE REVIEW

**Review by AlisaVlasova — CHANGES_REQUESTED** [review]
image
start-day modifier doesn't work
image

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,94 @@
...
+        50% {
+          border-radius: 50%;
+        }
+
+        100% {
+          border-radius: 0;
+        }
+      }
+
+      @keyframes color {
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss]: just add transition to `color`, you don't need animation here

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,94 @@
...
+      &:nth-child(#{$count-days})::before {
+        content: "#{$count-days}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translateY(-20px);
+      animation: round 0.5s, color 0.5s;
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss]: it's too much on hover)

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,3 +1,94 @@
...
+    font-size: 30px;
+
+    @for $count-days from 1 through 31 {
+      &:nth-child(#{$count-days})::before {
+        content: "#{$count-days}";
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss]: add this to `&__cell`

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,94 @@
...
+  flex-wrap: wrap;
+  padding: $calender-padding;
+  gap: $cell-gap;
+  width: ($cell-width * 7) + ($cell-gap * 6) + ($calender-padding * 2);
+
+  &--position {
+    position: absolute;
+    top: $calender-positioning;
+    left: $calender-positioning;
+    transform: translate(-$calender-positioning, -$calender-positioning);
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss]: use `-50%`

File: src/styles/main.scss [hunk 5]
```diff
@@ -1,3 +1,94 @@
...
+  }
+
+  $days:
+    "sun" ($cell-width * 6) + ($cell-gap * 5),
+    "sat" ($cell-width * 5) + ($cell-gap * 4),
+    "fri" ($cell-width * 4) + ($cell-gap * 3),
+    "thu" ($cell-width * 3) + ($cell-gap * 2),
+    "wed" ($cell-width * 2) + ($cell-gap * 1),
+    "tue" $cell-width,
+    "mon" 0;
```
Comment (ivanchuk-yurii) [inline src/styles/main.scss]: calculate values in loop

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]
modifier works now, but it should display from left side
image

File: src/styles/main.scss (hunk 2 above)
Comment (Viktor-Kost) [reply src/styles/main.scss]: is not fixed. Just change your color transition to 250ms

**Review by Viktor-Kost — APPROVED** [review]
ok!)

just apply previous mentor recommendation. A left an unresolved conversation for you to find

**Pull request closed without merging**